tests/molecules/C4_ae/CMakeLists.txt

```diff
@@ -0,0 +1,407 @@
+IF (NOT QMC_CUDA)
+ IF (NOT QMC_COMPLEX)
+
+# -150.95531020041986
+
+#  Reference from VMC runs
+#  LIST(APPEND C4_SD_AE_NO_SHORT "kinetic" "150.65661743 2.74357684")
+#  LIST(APPEND C4_SD_AE_NO_SHORT "totenergy" "-150.80373859 0.07367645") 
+#  LIST(APPEND C4_SD_AE_NO_SHORT "potential" "-301.46035602 2.74823426")
+#  LIST(APPEND C4_SD_AE_NO_SHORT "eeenergy" "106.91828633 0.08982962") 
+#  LIST(APPEND C4_SD_AE_NO_SHORT "ionion" "56.96059899 0.0003")
+#  LIST(APPEND C4_SD_AE_NO_SHORT "samples" "128000 0.0") 
+
+  LIST(APPEND C4_SD_AE_NO_SHORT "totenergy" "-150.95310200 0.07367645")
```

Comment:
Is the reference supposed to be -150.95310200 (as listed here) or  -150.95531020041986 (as listed above)